src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss

```diff
@@ -0,0 +1,113 @@
+@import '../../Media/Media.scss';
+@import '../../Variables.scss';
+
+$phone-range-small-end: 340px;
+$tablet-range-small-end: 670px;
+
+$navbar-border: solid 1px var(--GrayStrokeAndDisabled--translucent);
+
+.hamburger {
+  position: fixed;
+  top: var(--Space-24);
+  right: var(--Space-24);
+  z-index: $z-index-modal;
+
+  @include for-laptop-and-up {
+    display: none;
+  }
+}
+
+.mobileNavbar {
+  position: relative;
+  height: var(--Space-64);
+  background-color: var(--White);
+  padding: var(--Space-24) var(--ContainerPadding--Phone);
+  display: none;
+  align-items: center;
+  justify-content: flex-end;
+  display: flex;
+  @include for-laptop-and-up {
+    display: none;
+  }
+  &.visible {
+    z-index: $z-index-navbar-outer;
+  }
+  .hamburger {
+    position: absolute;
+  }
+
+  :global {
```

Comment:
Can we comment for why we need the `:global`

Reply:
4d77a30